### PR TITLE
15130-update-chapter31-address-schema

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
Corresponds to this PR into vets-json-schema - https://github.com/department-of-veterans-affairs/vets-json-schema/pull/535

This pull request updates the schema import for chapter31 form 28-1900. 

## Testing done
- local, all tests pass, forms behave as they should

## Screenshots
<details><summary>Form States</summary>

Screen Shot 2020-10-22 at 10 26 57 AM
Screen Shot 2020-10-22 at 10 27 18 AM
Screen Shot 2020-10-22 at 10 27 50 AM
Screen Shot 2020-10-22 at 10 55 12 AM
Screen Shot 2020-10-22 at 10 55 29 AM
Screen Shot 2020-10-22 at 10 55 56 AM

</details>

## Acceptance criteria
- [x] tests pass
- [x] form behaves as expected 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -341,7 +341,7 @@
     "url-search-params-polyfill": "^8.1.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#0aea5769bad46b9da4e60f8d40c613ad7a1c6bad",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#d8a039099a64a198c45280bf4ca83cabd2a2968d",
```
Comment (micahchiang) [inline package.json:344]: This hash will be updated to the one generated when [this pr](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/535) is merged into master.

File: src/applications/vre/28-1900/config/chapters/veteran/veteran-information/veteranInformation.js [hunk 2]
```diff
@@ -41,7 +42,14 @@ export const uiSchema = {
       ...ssnUI,
     },
     vaFileNumber: {
-      'ui:title': 'Your VA file number (*If different from SSN)',
+      'ui:title': (
+        <p className="vads-u-margin--0 vads-u-margin-top--3 vads-u-display--inline-block vads-u-font-weight--normal vads-u-color--base vads-u-font-family--sans vads-u-font-size--base">
+          Your VA file number{' '}
+          <span className="schemaform-required-span">
+            (*If different from SSN)
```
Comment (Mottie) [inline src/applications/vre/28-1900/config/chapters/veteran/veteran-information/veteranInformation.js:49]: Does the `SSN` abbreviation need to be spelled out? Something like:

```html
<dfn>
  <abbr title="social security number">SSN</abbr>
</dfn>
```

**Review by Mottie — APPROVED** [review]

File: src/applications/vre/28-1900/config/chapters/veteran/veteran-information/veteranInformation.js (hunk 2 above)
Comment (jenstrickland) [reply src/applications/vre/28-1900/config/chapters/veteran/veteran-information/veteranInformation.js]: This is correct. Code would be:

```
<span className="schemaform-required-span">
            (*If different from <abbr title="social security number">SSN</abbr>)
```

**Review by cohnjesse — APPROVED** [review]
At some point, we should update the standard address prefill so that it automatically pulls if this is a military address or not [here](https://github.com/department-of-veterans-affairs/vets-api/blob/03c6fb047e814f179c04e1e36a0786a3a0dad4a3/app/models/form_profile.rb#L48)

**Review by rianfowler — APPROVED** [review]

**Pull request merged**